<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(59, 59, 59)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>801</width>
      <height>661</height>
     </rect>
    </property>
    <widget class="QWidget" name="loginPage">
     <widget class="QFrame" name="loginframe">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>421</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QFrame#loginframe {
    background-color: #2e3338;
    padding: 20px;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="welcome_text">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 51, 56)</string>
       </property>
       <property name="text">
        <string>Welcome!</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="userinput">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="passinput">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>140</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="btnLogin">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>220</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Log In</string>
       </property>
      </widget>
      <widget class="QLabel" name="username_text">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 51, 56)</string>
       </property>
       <property name="text">
        <string>USERNAME</string>
       </property>
      </widget>
      <widget class="QLabel" name="password_text">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>120</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 51, 56)</string>
       </property>
       <property name="text">
        <string>PASSWORD</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="users_menu">
     <widget class="QGraphicsView" name="usMap">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>771</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>561</y>
        <width>771</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="welcomeUsers">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Welcome</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logoutBtn1">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="admin_menu">
     <widget class="QPushButton" name="usersMenuBtn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Plan a trip!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editBtn">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit Info</string>
      </property>
     </widget>
     <widget class="QLabel" name="welcomeAdmin">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Welcome admin!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logoutBtn2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="editInfo_menu">
     <widget class="QPushButton" name="goBackBtn">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logoutBtn3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
